.changeset/thirty-pets-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"ledger-live-desktop": patch
+---
+
+Hide tokens feature doesnt seem to work with some tokens
```

Comment:
I'd write: `Fixing "hide tokens" features that doesnt seem to work with some tokens`